app/views/planning_applications/submit_recommendation.html.erb

```diff
@@ -1,43 +1,57 @@
 <% content_for :page_title do %>
   Submit recommendation - <%= t('page_title') %>
 <% end %>
-
-<% content_for :title, "Submit recommendation" %>
-
+<% render(
+  partial: "shared/assessment_task_breadcrumbs",
+  locals: {
+    planning_application: @planning_application,
+    current_page: t(".review_and_submit_recommendation")
+  }
+) %>
 <%= render(
-  layout: "shared/assessment_dashboard",
```

Comment:
NB We're not only using `"shared/assessment_dashboard"` in four places. Everywhere else we're just using `"shared/proposal_header"`. The only additional thing `"shared/assessment_dashboard"` has is some optional warnings which should probably only display on the planning application show page anyway.

So future refactor to get rid of `"shared/assessment_dashboard"` altogether!